source/2020-08-25-accessibility-working-group-update.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Accessibility Working Group Update
+author: Melanie Sumner
+tags: Recent Posts, A11y, Accessibility, 2020
+responsive: true
+---
+
+Back in March, an Accessibility(A11y) Strike Team was formed to address the issues outlined in [Ember RFC Issue 595- Technical Accessibility Issues in New Ember Apps](https://github.com/emberjs/rfcs/issues/595). This blog post is to update the community on that group's work. The Strike Team met weekly from [March 25th](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-03/march-25.md) to [June 24th](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-06/june-24.md). We even managed to celebrate [Global Accessibility Awareness Day](https://blog.emberjs.com/2020/05/21/global-accessibility-awareness-day.html) somewhere in-between!
+
+READMORE
+
+There were five items on the list: 
```

Comment:
This sentence sounded a bit awkward to me (what list mentioned before?). Maybe you were thinking of:

```suggestion
The Strike Team is working on 5 items.
```

?

Reply:
```suggestion
There were five items listed in [Issue 595](https://github.com/emberjs/rfcs/issues/595): 
```

Reply:
Ah, perfect!